queries/commonlisp/highlights.scm

```diff
@@ -156,22 +125,11 @@
 
```

Comment:
pushed those all into a single line. since the list is auto-generated, I don't expect any value of reading it and instead let it occupy the least screen space possible.